sogs/model/room.py

```diff
@@ -529,16 +529,22 @@ def get_messages_for(
         recent: bool = False,
         single: Optional[int] = None,
         limit: int = 256,
+        reactions: bool = True,
+        reaction_updates: bool = True,
+        reactors: int = 0,
```

Comment:
`reaction_fetch_limit` instead of `reactors`?

Reply:
Renamed to `reactor_limit`